src/arm/BONE-UART5-00A0.dts

```diff
@@ -23,5 +26,8 @@
 };
 
 &bone_uart_5 {
+    // See these files for more info
+    // https://github.com/beagleboard/BeagleBoard-DeviceTrees/src/arm/bbai-bone-buses.dtsi
+    // https://github.com/beagleboard/BeagleBoard-DeviceTrees/src/arm/bbb-bone-buses.dtsi
     status = "okay";
```

Comment:
Is this acceptable?

Reply:
Instead of "for more info", how about "for the definition". They will live somewhere else upstream.

Reply:
Completed this :)